### ISSUE TITLE
Update to 0.2.4 breaks Nextcloud with database error "redirectUri is not a valid attribute"

### ISSUE THREAD

**Issue by Larx** [H2CK/oidc#66]
I run Nextcloud 24.0.6 with PostgreSQL. The update broke the installation and I had to disable the app:

```
Database error when running migration 0003Date20220927082100 for app oidc
redirectUri is not a valid attribute
```

**Comment by H2CK** [comment]
This error occurred with version 0.2.2. I assume you still have this version. Please check if you have really installed version 0.2.4. Otherwise install version 0.2.4 which fixes this problem.

**Comment by Larx** [comment]
I experienced this issue when upgrading to 0.2.4 IMHO (did not really take notice of version numbers, however). I than had to disable this app and was not able to re-enable it.
When looking in the list of disabled apps from the Nextcloud Web UI, however, I realized that I indeed had still installed 0.2.2. Updating that version from the list of disabled apps to 0.2.4 run flawlessly and now I was able to re-enable the app.
